Fix failing GitHub Actions CI/CD pipeline

## Summary
GitHub Actions workflow is failing on the `main` branch. The CI/CD pipeline needs to be fixed to allow successful builds and deployments.

## User Story
As a developer, I want the CI/CD pipeline to pass successfully, so that code changes can be automatically built, tested, and deployed.

## Context
- Current state: GitHub Actions workflow fails on every push to `main`
- Already fixed: Updated `DOTNET_VERSION` from `9.0.x` to `10.0.x` in `.github/workflows/deploy.yml` (commit `72751f7`)
- Already fixed: Updated `SummarizationSettingsTests` to expect `MaxTokens=500` instead of `150` (commit `9f34d52`)
- Problem: Build still fails - need to investigate the actual failure reason

## Requirements
### Must Have
- [ ] Investigate the actual failure in GitHub Actions run #20163400684
- [ ] Fix all failing tests
- [ ] Ensure `dotnet build` passes
- [ ] Ensure `dotnet test` passes (all 205+ tests)
- [ ] GitHub Actions workflow completes successfully

### Should Have
- [ ] Address NuGet warning NU1608 about EFCore.SqlServer.VectorSearch version mismatch

## Acceptance Criteria
- [ ] Given a push to `main`, when GitHub Actions runs, then all jobs complete successfully (green checkmark)
- [ ] All unit tests pass locally with `dotnet test`

## Technical Notes
- Workflow file: `.github/workflows/deploy.yml`
- Run logs: `gh run view 20163400684 --repo Olbrasoft/GitHub.Issues --log-failed`
- Local tests pass but CI might have different environment

## Out of Scope
- Adding new features
- Refactoring unrelated code

--- a/test/Olbrasoft.Text.Translation.DeepL.Tests/DeepLTranslatorTests.cs
+++ b/test/Olbrasoft.Text.Translation.DeepL.Tests/DeepLTranslatorTests.cs
@@ -117,7 +117,7 @@ public class DeepLTranslatorTests
 
         // Assert
         Assert.False(result.Success);
-        Assert.Contains("401", result.Error);
+        Assert.Contains("Unauthorized", result.Error);
         Assert.Equal("DeepL", result.Provider);
     }
 }

